Developer Samples/WindowsContainers/inrule-catalog-manager/Start.ps1

```diff
@@ -1,6 +1,7 @@
+write-host "Container IP address: $(get-netipaddress)"
 $last = Get-Date
 while ($true) {
-    start-sleep -seconds 2;
-    Get-WinEvent -FilterHashtable @{ StartTime = $last; } -ErrorAction SilentlyContinue | format-list
+    start-sleep -seconds 5;
+    (Get-WinEvent -FilterHashtable @{ StartTime = $last; ProviderName = "InRule*" } -ErrorAction SilentlyContinue) | format-list
```

Comment:
This may result in never seeing any logs being returned; the web catalog manager doesn't seem to always write to the InRule log when it has a problem (e.g., if the catalog service is unreachable).

Reply:
Hi @jelster

That was to fix this error that occurs otherwise:
```
Get-WinEvent : You must specify at least one Log, Provider or Path key-value pair.
At line:1 char:2
+ (Get-WinEvent -FilterHashtable @{ StartTime = Get-Date } -ErrorAction ...
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-WinEvent], Exception
    + FullyQualifiedErrorId : LogProviderOrPathNeeded,Microsoft.PowerShell.Commands.GetWinEventCommand
```

Reply:
That said, I haven't actually got the catalog manager container working correctly - I just get a blank page when I navigate to it, and I haven't yet dug into that any further.

Reply:
Ahhh thanks for the update. That makes sense. I must have copy/pasted the event log script into webcatman and modified it without thinking. 

I'll perform a test of the webcatman image to see if I also have issues with it.

Reply:
just did a quick test, I did not have any issues starting a new webcatman image pointing at a pre-existing (non-docker) catalog service. I did notice that the first time I brought up the site it did take about a few seconds to show up (15-30), before that it just looked like a blank page. The PR looks good to me!

Reply:
Have managed to get the catalog manager working now (after getting access to the ELMAH log).

It was failing due to the `App_Data\CatalogManager.sdf` file already existing in the directory that was copied into the docker image, and IIS not having write access to it. Creating an image without the database file present solves the problem.